pep-0722.rst

```diff
@@ -108,49 +157,50 @@ Backwards Compatibility
 For preexisting code this should require no changes, but should simplify this
 use case here:
 
-:: 
+.. code-block:: python
 
-    A = TypeVar("A")
-    B = TypeVar("B")
+    class A():
+        pass
+    class B():
+        pass
 
     def is_A(x: A | B) -> TypeGuard[A]:
-        raise NotImplementedError
+        return is_instance(x, A)
 
 
-    def after_is_A(x: A | B) -> TypeGuard[B]:
-        return True
+    def is_B(x: A | B) -> TypeGuard[B]:
+        return is_instance(x, B)
 
 
     def test(x: A | B):
         if is_A(x):
-            reveal_type(x)
+            # Do stuff assuming x is an 'A'
             return
-        assert after_is_A(x)
+        assert is_B(x)
 
-        reveal_type(x)
+        # Do stuff assuming x is a 'B'
         return
 
-["after_is_A" is confusing me -- is there a better name? "is_not_A"?]
-[Can/should you use PEP 695 syntax for the TypeVars?]
 
-becomes this instead
-["becomes this instead" is not a grammatically correct continuation of the sentence before the first code block. Maybe rephrase the sentence to "Preexisting code should require no changes, but code like this...can be simplified to this:"]
-[Add comments in these code blocks showing the expected inferred type as you did above? I think then you won't need the reveal_type calls?]
+This use case becomes this instead:
```

Comment:
With this proposed change, the code above continues to work but could be simplified by removing the assertion that `x` is of type `B` in the negative case: